app/templates/index.html

```diff
@@ -2,7 +2,17 @@
 {% block title %}Home{% endblock %}
 {% block body %}
 <div class="jumbotron">
-  <h1>Flask Is Awesome</h1>
-  <p class="lead">And I'm glad to be learning so much about it!</p>
+    <h1>What is Genten Studios?</h1>
+    <p class="lead">Genten Studios is a group of programmers who are passionate about creating games. All of our games
+        follow a common pattern that the program itself does not provide any content. Content is defined by modules
```

Comment:
"program itself does not provide any content" may wanna point out the lua interface API.
So maybe like "program itself does not provide any content other then a Lua API.

Reply:
Is a Lua API content? or is it an API?

Reply:
@apachano It's an API but it is in phoenix/quartz by default. 
Maybe "program itself does not provide any content but it has a Lua feature rich API that can be used to make all kinds of content." ?

Reply:
So the sentence following this one?

Reply:
I suppose. You can drop this one.